com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -16,6 +16,16 @@ namespace UnityEditor.VFX.UI
     [Serializable]
     class VFXViewWindow : EditorWindow
     {
+        private static Dictionary<Tuple<Type, bool>, string> vfxIconMap = new()
+        {
+            { new Tuple<Type, bool>(typeof(VisualEffectSubgraphOperator), true), "d_subgraph-operator.png" },
+            { new Tuple<Type, bool>(typeof(VisualEffectSubgraphBlock), true), "d_subgraph-block.png" },
+            { new Tuple<Type, bool>(typeof(VFXGraph), true), "vfx_graph_icon_gray_dark.png" },
+            { new Tuple<Type, bool>(typeof(VisualEffectSubgraphOperator), false), "subgraph-operator.png" },
+            { new Tuple<Type, bool>(typeof(VisualEffectSubgraphBlock), false), "subgraph-block.png" },
+            { new Tuple<Type, bool>(typeof(VFXGraph), false), "vfx_graph_icon_gray_light.png" },
+        };
+
```

Comment:
Anonymous type FTW \o/
```
private static Dictionary<(Type type, bool proSkin), string> vfxIconMap = new()
{
	{ (typeof(VisualEffectSubgraphOperator), true), "d_subgraph-operator.png" },
	{ (typeof(VisualEffectSubgraphBlock), true), "d_subgraph-block.png" },
	{ (typeof(VFXGraph), true), "vfx_graph_icon_gray_dark.png" },
	{ (typeof(VisualEffectSubgraphOperator), false), "subgraph-operator.png" },
	{ (typeof(VisualEffectSubgraphBlock), false), "subgraph-block.png" },
	{ (typeof(VFXGraph), false), "vfx_graph_icon_gray_light.png" },
};
```
But I'm not sure it works like Tuple when it's used as key 🤔 Let me check ⏳

Reply:
Nice, it seems to works as expected (see [here](https://dotnetfiddle.net/4gVftw))
![image](https://user-images.githubusercontent.com/22494706/148359231-7648cea8-86bb-4238-b4d9-204b9bd9098d.png)

Reply:
So, if you want to apply this change, use `Dictionary<(Type type, bool proSkin), string>` (which can be also readonly btw) and remove occurence of `new Tuple<Type, bool>` 🎉

Reply:
Nice 👍